### PR TITLE
Removes free roundstart RCDs, adds foam grenades to EVA

### PR DESCRIPTION
**Pull request by kriskog**
## About The Pull Request
Right. Since RCD's are now in the advanced engineering tech node, I would rather see people print their RCD instead of grabbing a free one from the vendor. That adds some pressure to get the node (its usually grabbed early anyway) and a not insignificant resource cost. Its moved to premium at its old cost of 1700 cr with 2 stock. The EVA RCD has been removed, and replaced with four smart metal foam grenades.

Foam has been changed to use reachableAdjacentTurfs to find neighbors as the old one didnt work on space. Should make the metal foam more reliable.

Oh and Kilostation did a LOT of key cleanup when run in SDMM. It was also missing from the maps.txt config on master, making it unable to be run. Also commented out the default value from box, we dont use default on the servers anymore.

## Why It's Good For The Game
Less free roundstart gamer gear.

## Changelog
:cl: Skoglol
balance: EVA RCD replaced with four smart metal foam grenades.
tweak: Foam now spreads properly on space tiles.
balance: Engineering vendor RCD's are now premium at 1700 cr with 2 stock. Reminder that these can be printed in the engineering lathe with the relevant tech.
/:cl:

### CODE REVIEW

**Comment by Ratvar** [comment]
What about removing autolathe RCD?

**Comment by JMoldy** [comment]
As an engineer main on Sybil: If you're going to do this, may I recommend making them cheaper in the protolathe as compensation? As it is, their exorbitant metal cost at the protolathe makes them unfeasible to print out enough to get Engineering well equipped without sucking away the entire iron budget of the station. You may also want to consider how this affects the fact that they can be produced at hacked autolathes. 
It'd also be nice to get a cost of 1500 instead of 1700.

**Comment by cacogen** [comment]
I thought it was removed from the hacked autolathe and placed in the supply departmental lathe? Also I hate showing up late as an engineer and all the RCDs are already gone from the vendor so I'd like if it took a bit longer for engineers to have the necessary means the necessary means to buy them (engineers start with a lot of credits anyway so it probably won't make a difference).

**Comment by kriskog** [comment]
The autolathe RCD was removed in #47289, in favor of the techweb one.
A full RCD in a t1 lathe (you got t2 at least at this point, use them) costs 54000 iron, 4500 glass. Or 26 metal and 2.25 glass. t4 is less than half. Engineering alone has 3-4 stacks worth of glass and metal roundstart, you'll be fine.

**Comment by CRITAWAKETS** [comment]
Making something unavailable to a job roundstart which is something that a powergamer tides for every round is a really, really bad idea. Expect the captain, a traitor with money briefcases or security to just take all the RCDs for themselves.

**Comment by Krysonism** [comment]
>but assistants don't need it.

Assistants really appreciate the RCD for both breaking & entering and for building bases.

Maybe just move the RCD crate to the vault to make it more risky to steal?

**Comment by kriskog** [comment]
It's locked behind advanced engineering, or if you are unfamiliar with it, tech thats usually taken in the first 10 minutes out of pure necessity. Engineers can wait the extra 10 minutes, or pool their cash if they desperately need one roundstart. Mind you, not even command has enough credits on their own at roundstart to buy one.

We don't code for assistants, they aren't a real job. If they want something to make their projects easier, they can work for it like the rest of us.

**Comment by Iamgoofball** [comment]
RCDs are a traitor theft item, there needs to be at least one.

**Comment by kriskog** [comment]
Its not. When did you even play the game last?

**Comment by Iamgoofball** [comment]
Like a year? Although I haven't rolled steal objective in like 2 years, shits rare. Why'd we remove rcd theft?

**Comment by Shadowflame909** [comment]
This does make those space ruins harder. IE those syndicate ones with the bolted shocked doors, and lasers everywhere.

**Comment by kriskog** [comment]
> 
> 
> Like a year? Although I haven't rolled steal objective in like 2 years, shits rare. Why'd we remove rcd theft?

Okay, that wasnt entirely fair of me. Not sure, but I haven't ever seen it and I started around april 2018. I assume they were too common? Its just an autolathe (which were public at the time) and a bit of metal.

>
>
> This does make those space ruins harder. IE those syndicate ones with the bolted shocked doors, and lasers everywhere.

Good.

**Comment by Shadowflame909** [comment]
Why good? I assume that's why it's there. And those ruins are designed around it being there.

I dunno, removing it because of

>Less free roundstart gamer gear.

Doesn't hold up if you're complaining about assistants breaking and entering into places. Which is a sec/administration issue.

Tldr; Doesn't stop tiding. Does harm those situations when this item is necessary, like space exploring though. Or the malf ai round.

**Comment by zxaber** [comment]
I've looted every space ruin that's worth looting and ever bothered to have an RCD. The syndicate ship has titanium walls which are just a welder and wrench away from breaching, it's not a big deal. The turrets don't hit you if you break in near the thrusters.

Out of scope for this PR, but maybe a rudimentary round-start RCD that can only place and destroy foam walls/floors would be neat. It would allow quick patch-ups without allowing easy break-ins. The vendor RCDs could be replaced with this as a non-premium option.

**Comment by Shadowflame909** [comment]
>break the walls bro

I am at least 90% sure that the space hotel ruin has unbreakable walls that require the RCD to destroy that bolted door.

RCD is old tier, because we still got some old ruins in service.

**Comment by kriskog** [comment]
Bring hacking tools :confused:

**Comment by Geikkamir** [comment]
Haven't people been saying that construction needs to be _easier_ and _less_ time consuming?

**Comment by cacogen** [comment]
the answer to that is to add a skill that engineers start with that nerfs build time/material costs when building by hand

**Comment by zxaber** [comment]
Advanced engineering RPDs are hardly even mid-game tech. You should have them researched and available within like 15 minutes of a round.

**Comment by Tlaltecuhtli** [comment]
>vendor stock down to 2

why? 😂

**Comment by Mickyan** [comment]
skogineering

May be out of scope for this but putting a few foam grenades in the premium section of the public tool vendor wouldn't be the worst idea

**Comment by JMoldy** [comment]
While I don't think anyone is terribly bothered about the roundstart EVA RCD's, ensuring that roundstart engineers can't get them at roundstart is going to turn every early-game explosion into a huge pain to repair. Medbay already gets wrecked often enough roundstart, forcing engineers to manually fix it is just going to turn it into a crater for the entirety of many rounds.

**Comment by ExcessiveUseOfCobblestone** [comment]
conflicts
